src/Models/CompanyBan.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace TruckersMP\APIClient\Models;
+
+use Carbon\Carbon;
+
+class CompanyBan
+{
+    /**
+     * The ID of the ban.
+     *
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * The ID of the user.
+     *
+     * @var int
+     */
+    protected $userId;
+
+    /**
+     * The username of the user.
+     *
+     * @var string
+     */
+    protected $username;
+
+    /**
+     * The Steam ID of the user.
+     *
+     * @var int
+     */
+    protected $steamId;
+
+    /**
+     * The role ID of the user.
```

Comment:
It should suggest that it is the company's role, not the user group.

```suggestion
     * The player's role ID within the company.
```